assets/src/ba_data/python/bastd/actor/spazbot.py

```diff
@@ -256,7 +257,9 @@ def update_ai(self) -> None:
                 target_pt_raw = ba.Vec3(*self.target_flag.node.position)
                 diff = (target_pt_raw - our_pos)
                 diff = ba.Vec3(diff[0], 0, diff[2])  # don't care about y
-                dist = diff.length()
+                dist = distance(
+                    (target_pt_raw.x, 0, target_pt_raw.z),
+                    (our_pos.x, 0, our_pos.z))
```

Comment:
Doesn't the old code do the same as this? (it gets the difference between target_pt_raw and our_pos, zeroes y, and then gets the length of that).

I'd like to use the ba.Vec3 class when possible since that is implemented in C++ so should be more efficient. If the math is broken in any way or it needs more functionality, I'm happy to fix it.